python/doc/theory/reliability_sensitivity/taylor_moments.rst

```diff
@@ -1,144 +1,106 @@
+       {\partial x_i \partial x_j}\right)(\vect{\mu})
 
--  :math:`\Cov \uX` is the covariance matrix of the random vector
-   :math:`\uX`. The elements are the followings :
-   :math:`(\Cov \uX)_{ij} = \Expect{\left(X^i - \Expect{X^i} \right)^2}`
 
--  :math:`\underline{\underline{\nabla}} h(\muX) = \: ^t \left( \frac{\partial y^i}{\partial x^j}\right)_{\ux\: =\: \muX} = \: ^t \left( \frac{\partial h^i(\ux)}{\partial x^j}\right)_{\ux\: =\: \muX}`
-   is the transposed Jacobian matrix with :math:`i=1,\ldots,n_Y` and
-   :math:`j=1,\ldots,n_X`.
+The second order approximation of the variance  is not implemented because it requires both the knowledge of higher
+order derivatives of :math:`h` and the knowledge of moments of order strictly greater
+than 2 of the distribution of :math:`\uX`.
 
--  :math:`\underline{\underline{\underline{\nabla^2}}} h(\ux\:,\ux)` is
-   a tensor of order 3. It is composed by the second order derivative
-   towards the :math:`i^\textrm{th}` and :math:`j^\textrm{th}`
-   components of :math:`\ux` of the :math:`k^\textrm{th}` component of
-   the output vector :math:`h(\ux)`. It yields to:
-   :math:`\left( \nabla^2 h(\ux) \right)_{ijk} = \frac{\partial^2 (h^k(\ux))}{\partial x^i \partial x^j}`
 
--  :math:`<\underline{\underline{\nabla}}h(\muX) , \: \uX - \muX> = \sum_{j=1}^{n_X} \left( \frac{\partial {\uy}}{\partial {x^j}}\right)_{\ux = \muX} . \left( X^j-\muX^j \right)`
+**Case 2:** :math:`Y =(Y_1, \dots, Y_q) \in \Rset^q` with :math:`q>1`:
 
--  :math:`<<\underline{\underline{\underline{\nabla }}}^2 h(\muX,\: \underline{\mu}_{X}),\: \uX - \muX>,\: \uX - \muX> =
-   \left( ^t (\uX^i - \muX^i). \left(\frac{\partial^2 y^k}{\partial x^i \partial x^k}\right)_{\ux = \muX}. (\uX^j - \muX^j) \right)_{ijk}`
 
-| **Approximation at the order 1 - Case** :math:`n_Y>1`
+The second order Taylor expansion of :math:`h = (h_1, \dots, h_q)` around :math:`\ux = \vect{\mu}` is written
+for each marginal function :math:`h_k` as:
 
-  .. math::
+.. math::
 
-       \Expect{\uY} \approx \underline{h}(\muX)
+      Y_k = h_k(\vect{\mu}) + \sum_{i = 1}^d \left( \frac{\partial h_k}{\partial x_i }\right)(\vect{\mu})
+      (X_i-\mu_i)+ \frac{1}{2} \sum_{i,j = 1}^d \left( \frac{\partial^2 h_k}{\partial x_i \partial
+      x_j}\right)(\vect{\mu})(X_i-\mu_i)(X_j-\mu_j) + o(||\uX||^2)
 
-  Pay attention that :math:`\Expect{\uY}` is a vector. The
-  :math:`k^\textrm{th}` component of this vector is equal to the
-  :math:`k^\textrm{th}` component of the output vector computed by the
-  model :math:`h` at the mean value. :math:`\Expect{\uY}` is thus the
-  computation of the model at mean.
 
-.. math::
+where :math:`1\leq k \leq q`.
 
-    \Cov \uY \approx ^t\underline{\underline{\nabla}}\:\underline{h}(\muX).\Cov \uX.\underline{\underline{\nabla}}\:\underline{h}(\muX)
+If we use the first order expansion, we get:
 
-| **Approximation at the order 2 - Case** :math:`n_Y>1`
 
-.. math::
+  .. math::
+
+       \begin{array}{lcl}
```

Comment:
Please remove the unnecessary array here.

Reply:
ok. I did it on the entire file